web/app/kela/KelaVersiohistoria.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'baret'
+import Atom from 'bacon.atom'
+import Http from '../util/http'
+import {ISO2FinnishDateTime} from '../date/date'
+import Text from '../i18n/Text'
+import Link from '../components/Link'
+
+export const KelaVersiohistoria = ({opiskeluoikeus, oppijaOid}) => {
+  const opiskeluoikeudenOid = opiskeluoikeus.oid
+  const versionumero = opiskeluoikeus.versionumero
+  if (!(opiskeluoikeudenOid || versionumero)) return null
+
+  const historyObs = Http.cachedGet(`/koski/api/luovutuspalvelu/kela/versiohistoria/${opiskeluoikeudenOid}`)
+  const showHistoryAtom = Atom(false)
+
+  return (
+    <div className='versiohistoria'>
+      {historyObs.combine(showHistoryAtom, (history, showHistory) => {
+        return (<>
+          <span onClick={() => showHistoryAtom.set(!showHistory)}><Text name='Versiohistoria'/></span>
+          {showHistory && (
+            <div className='kela-modal'>
+              <div className='kela-modal-content'>
+              <button onClick={() => showHistoryAtom.set(false)} className='kela-content-close'/>
+              <ol>{history.map((version, i) =>
```

Comment:
WARNING in ./app/kela/KelaVersiohistoria.jsx
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/ahtiainen/work/oph/koski/web/app/kela/KelaVersiohistoria.jsx
  26:19  warning  Missing parentheses around multilines JSX  react/jsx-wrap-multilines

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.